src/state/application/hooks.ts

```diff
@@ -121,6 +121,10 @@ export function useNotificationModalToggle(): () => void {
   return useToggleModal(ApplicationModal.NOTIFICATION_SUBSCRIPTION)
 }
 
+export function useEligibleTxToggle(): () => void {
+  return useToggleModal(ApplicationModal.ELIGIBLE_TX)
+}
```

Comment:
dont need to manage it as global state, move into local state of tx modal please

Reply:
Do you have any example of local modal?

Reply:
TutorialModal